### PR TITLE
Droth 1358 notification of production updates

### PR DESCRIPTION
**Pull request by joanacoelho**

### CODE REVIEW

**Comment by oskarmalmstrom** [comment]
ok

File: UI/src/view/userNotificationPopup.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+     '<div class="modal-overlay confirm-modal" id="notification">' +
+     '<div class="modal-dialog notification">' +
+     '<div class="content">' + options.message + '</div>' +
+       me.createNotificationForm(notifications) +
+     '<div class="actions">' +
+     '<button class = "btn btn-secondary cancel">' + options.cancelButton + '</button>' +
+     '</div>' +
+     '</div>' +
+     '</div>';
+   };
```
Comment (MAC-CGI) [inline UI/src/view/userNotificationPopup.js]: Maybe we could create something like actionPanelBox.js

     '<div class="modal-overlay confirm-modal" id=" + me.popUpName() + '">' +
     '<div class="modal-dialog notification">' +
     '<div class="content">' + 
	 me.message() +
         me.close() +
	 + '</div>' +
        me.formContent() +
     '<div class="actions">' +
     me.buttons() +
     '</div>' +
     '</div>' +
     '</div>';

File: digiroad2-oracle/src/main/resources/db/migration/V172__create_notification_table.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+create table notification
+(
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/resources/db/migration/V172__create_notification_table.sql]: maybe the name could be change to user_notification

File: digiroad2-oracle/src/main/resources/db/migration/V172__create_notification_table.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  id number primary key,
+  created_date DATE default sysdate,
+  heading varchar2(256),
+  content varchar2(4000)
+);
+
+
+CREATE SEQUENCE notification_seq
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/resources/db/migration/V172__create_notification_table.sql]: could be rename to user_notification_seq

File: UI/src/less/site/modals.less [hunk 4]
```diff
@@ -65,6 +65,68 @@
...
+
+  .selected {
+    font-weight: bold;
+    font-size: 13px;
+  }
+
+  .bold {
+    font-weight: bold;
+  }
+
```
Comment (MAC-CGI) [inline UI/src/less/site/modals.less]: .bold and .selected should be inside .notification, **.modal-overlay.confirm-modal** is used in more places together

**Pull request merged**